CODEOWNERS

```diff
@@ -1316,6 +1316,7 @@ build.json @home-assistant/supervisor
 /tests/components/tasmota/ @emontnemery
 /homeassistant/components/tautulli/ @ludeeus @tkdrob
 /tests/components/tautulli/ @ludeeus @tkdrob
+/homeassistant/components/tedee/ @patrickhilker @zweckj
```

Comment:
Please run hassfest, you're not added as codeowner for the tests

Reply:
Huh. Shouldn't the pre-commit hooks do that.

Reply:
Yes, but I don't know why this happens